examples/minimalMilkdrop/README.md

```diff
@@ -0,0 +1,11 @@
+# Minimal Milkdrop Example
```

Comment:
Maybe add a note that this is not yet fully supported/released? That way we can merge now and hold off on the release until we get the rest of the stuff done.

Reply:
ah, good call!